server/tests/controllers/articles.js

```diff
@@ -63,18 +108,52 @@ describe('Articles controller', () => {
         .set('Content-Type', 'application/json')
         .send(Article)
         .end((err, res) => {
-          res.should.have.status(201);
-          res.body.should.be.a('object');
-          const { article } = res.body;
-          article.should.be.an('object');
-          article.should.have.property('slug');
-          article.should.have.property('title');
-          article.should.have.property('description');
-          article.should.have.property('body');
-          article.should.have.property('createdAt');
-          article.should.have.property('updatedAt');
-          article.author.should.be.a('object');
-          testSlug = article.slug;
+          res.should.have.status(200);
+          const { newArticleAlert } = res.body;
+          newArticleAlert.should.be.a('object');
+          newArticleAlert.createdArticle.should.be.an('object');
+          newArticleAlert.createdArticle.should.have.property('slug');
+          newArticleAlert.createdArticle.should.have.property('title');
+          newArticleAlert.createdArticle.should.have.property('description');
+          newArticleAlert.createdArticle.should.have.property('body');
+          newArticleAlert.createdArticle.should.have.property('tags');
+          newArticleAlert.createdArticle.should.have.property('createdAt');
+          newArticleAlert.createdArticle.should.have.property('updatedAt');
+          newArticleAlert.author.should.be.a('object');
+          testSlug = newArticleAlert.createdArticle.slug;
+          done();
+        });
+    });
+
+    it('should be able to create articles with tags', (done) => {
+      chai.request(server)
+        .post('/api/articles')
+        .set('Accept', 'application/json')
+        .set('Authorization', `Bearer ${token1}`)
+        .set('Content-Type', 'application/json')
+        .send(Article2)
+        .end((err, res) => {
+          res.should.have.status(200);
+          const { newArticleAlert } = res.body;
+          newArticleAlert.should.be.a('object');
+          newArticleAlert.createdArticle.should.be.an('object');
+          newArticleAlert.createdArticle.should.have.property('slug');
+          newArticleAlert.createdArticle.should.have.property('title');
+          newArticleAlert.createdArticle.should.have.property('description');
```

Comment:
@emekafredy I think you can make this code more consise by using destructuring to extract `createdArticle` from `newArticleAlert` as well. lines `114 - 121` and `139 - 142`

Reply:
Okay. I'm also working on this